app/src/main/java/at/roteskreuz/stopcorona/model/db/DefaultDatabase.kt

```diff
@@ -33,21 +31,20 @@ import at.roteskreuz.stopcorona.skeleton.core.model.db.converters.DateTimeConver
         DbDailyBatchPart::class,
         DbQuestionnaireAnswer::class,
         DbPageContent::class,
-        DbReceivedInfectionMessage::class,
         DbSentTemporaryExposureKeys::class
     ],
-    version = 21,
+    version = 22,
```

Comment:
Sure? Has already been increased

Reply:
Yes, the current strategy in order to avoid having un-migrated changes is to create a migration in each open task.  This way we can test the migrations also during the development process.

Reply:
Ok. Just told Falko not to do it :-)